### ISSUE TITLE
Cache Optionality

### ISSUE THREAD

**Issue by landmann** [unclecode/crawl4ai#137]
Allow the user to choose which folder to cache contents

**Comment by unclecode** [comment]
Hi, right now we create and use sqlite, and save the database in the home directory under the crawl4ai (e.g. `~/.crawl4ai`) folder. Mu understanding is you referring to allowing users to choose the home directory so the rest will be there. I think that would be better because there are other certain things that we save over there, including the caching database. Is this something you're asking for? @landmann

**Comment by landmann** [comment]
Yep! Essentially, we're offloading these to a lambda function, and the lambda has a shared storage volume where I want to put this sqlite - this volume is not in the home directory so I'd love to specify where to put this sqlite file. Hope this makes sense, and thank you for this great library @unclecode !!
